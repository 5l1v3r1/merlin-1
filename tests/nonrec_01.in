["reset","name","test.ml"]
["extension","enable",["nonrec"]]
["tell","source","type nonrec 'a t = 'a list\n"]
["tell","end"]
["errors"]
["dump","env"]
