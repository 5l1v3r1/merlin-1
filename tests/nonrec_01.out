["return",{"line":1,"col":0}]
["return",true]
["return",null]
["return",{"line":2,"col":0}]
["return",[]]
["return",[["Line 1, characters 12-27","type 'a t = 'a list"]]]
